docs/tutorials/control/flutter-app.md

```diff
@@ -422,6 +422,23 @@ Follow the steps below to get your API key and create an environment variables f
      }
    ```
 
+8. Reopen your <file>pubspec.yaml</file> file and paste the following two lines at the end of it, inside the `flutter:` section.
```

Comment:
(opt) You could probably combine this into just one instruction

Reply:
Sorry I don't understand. What do you mean?

Reply:
It's hard to understand where to put the first code block without the second, so you could probably just have the second code block!